resources/model/OpenStudio.idd

```diff
@@ -14695,7 +14695,8 @@ OS:GroundHeatExchanger:Vertical,
         \required-field
         \type object-list
         \object-list ConnectionNames
-  N1,   \field Maximum Flow Rate
+  N1,   \field Design Flow Rate
```

Comment:
I'm deleting field 11 (Design Flow Rate) and moving it in place of field 4 (Maximum Flow Rate)